io.github.Arian94.Tiny-beast-dictionary.yml

```diff
@@ -0,0 +1,31 @@
+app-id: io.github.Arian94.Tiny-beast-dictionary
+runtime: org.gnome.Platform
+runtime-version: '44'
+sdk: org.gnome.Sdk
+command: tiny-beast-dict
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --socket=pulseaudio
+  - --filesystem=home/.cache/${FLATPAK_ID}    #possibly needed if flatpak does not create local cache in app dir.
+  - --env=xsel=/app/bin/xsel
+modules:
+  - name: tiny-beast-dictionary
+    buildsystem: simple
+    build-commands:
+      - install -D src-tauri/tiny-beast-dictionary ${FLATPAK_DEST}/bin/tiny-beast-dictionary
+      - install -Dm644 src-tauri/tiny-beast-dictionary.desktop /app/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644 src-tauri/icons/32x32.png /app/share/icons/hicolor/32x32/apps/${FLATPAK_ID}.png
+      - install -Dm644 src-tauri/icons/128x128.png /app/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.png
+      - install -Dm644 ${FLATPAK_ID}.appdata.xml /app/share/appdata/${FLATPAK_ID}.appdata.xml
+      - install -Dm755 src-tauri/xsel /app/bin/xsel
```

Comment:
what will xsel do on wayland?

Reply:
I built my app on Wayland (maybe doesn't matter) and checked it and it works. I don't know why.